.pylintrc

```diff
@@ -135,7 +135,8 @@ disable=print-statement,
         exception-escape,
         comprehension-escape,
         unused-argument,
-        too-many-lines
+        too-many-lines,
+        protected-access
```

Comment:
add protected-access to pylintrc ignore list, because we access those methods in test scripts